deployments/bridges/rialto-millau/docker-compose.yml

```
@@ -0,0 +1,28 @@
+version: '3.5'
+services:
+  relay-millau-headers-to-rialto:
```

```suggestion
  relay-headers-millau2rialto
```
or `relay-headers-millau-rialto` would be my preference. I think we should also be consistent with naming now.